uamqp/message.py

```diff
@@ -70,6 +82,7 @@ def __init__(self,
                  message=None,
                  settler=None,
                  delivery_no=None,
+                 body_type=None,
```

Comment:
Wouldn't this be a breaking change to the constructor?

Reply:
good catch, yes, I should move this to the end of the constructor to avoid breaking change

Reply:
code updated